dev/bots/suite_runners/run_flutter_driver_android_tests.dart

```diff
@@ -79,4 +79,49 @@ Future<void> runFlutterDriverAndroidTests() async {
       'native_driver_test',
     ),
   );
+
+  // A simulated cutout overlay is necessary to test display_cutout_rotation.
```

Comment:
This has the possibility of polluting the test device, and/or making future tests added (or refactored) have confusing behavior. That being said, that's sort of true of _all_ `adb` commands, but at least it would be more obvious.

Let's make sure we toggle these settings off. I'd like to see a pattern like:

```dart
try {
  _enableDevelopmentSettings();
  _enableOverlay('com.android.internal.display.cutout.emulation.tall');
  runCommand('flutter', ['drive', ...]);
} finally {
  _disableDevelopmentSettings();
  _disableOverlay();
}
```